packages/rancher-monitoring/generated-changes/overlay/files/rancher/home/rancher-default-home.json

```diff
@@ -1,1275 +1,1273 @@
 {
-    "annotations": {
```

Comment:
Updated to contain everything that rancher/home/rancher-default-home-with-windows.json previously separately maintained and heavily refactored.